Gemfile

```diff
@@ -21,6 +21,7 @@ gem 'hashie'
 gem 'hiredis'
 gem 'http_accept_language'
 gem 'httparty'
+gem 'identity-hostdata', github: '18F/identity-hostdata', branch: 'margolis-initial-metadata'
```

Comment:
please review this PR so I can merge this to master first! https://github.com/18F/identity-hostdata/pull/1

Reply:
👍